src/gui/widgets/LcdWidget.cpp

```diff
@@ -94,6 +94,22 @@ void LcdWidget::setValue(int value)
 	update();
 }
 
+void LcdWidget::setValue(float value)
```

Comment:
This is a good way of fixing the minus sign issue. Better than what I came up with